src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -221,6 +224,10 @@ def setUp(self):
                     "Test requires {} {}; {} specified".format(
                         expected_count, host_type, actual_count))
 
+        #Storage setup if requested in test input file
+        if self.nvme_parameter == "nvme":
+            server_utils.storage_prepare(self.hostlist_servers)
```

Comment:
Could you please add a condition if it is something other than nvme.

Reply:
Please refer new PR#1067. I am going to close this one.

Right now we have nvme only tested so later when we tested with other option we can add those options.